src/components/WalletInit/ConnectNanoX/CheckNanoXScreen.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import React from 'react'
+import {View, SafeAreaView, ScrollView, Image, Platform} from 'react-native'
+import {injectIntl, defineMessages} from 'react-intl'
+import {compose} from 'redux'
+import {withHandlers} from 'recompose'
+
+import {Text, Button, Link, BulletPointItem, ProgressStep} from '../../UiKit'
+import {withNavigationTitle} from '../../../utils/renderUtils'
+import {
+  confirmationMessages,
+  ledgerMessages,
+} from '../../../../src/i18n/global-messages'
+import {WALLET_INIT_ROUTES} from '../../../RoutesList'
+
+import styles from './styles/CheckNanoXScreen.style'
+import image from '../../../assets/img/ledger_1.png'
+
+import type {ComponentType} from 'react'
+import type {IntlShape} from 'react-intl'
+import type {Navigation} from '../../../types/navigation'
+
+const messages = defineMessages({
+  title: {
+    id: 'components.walletinit.connectnanox.checknanoxscreen.title',
+    defaultMessage: '!!!Connect to Ledger Nano X',
+  },
+  introline: {
+    id: 'components.walletinit.connectnanox.checknanoxscreen.introline',
+    defaultMessage: '!!!Before continuing, please make sure that:',
+  },
+  learnMore: {
+    id: 'components.walletinit.connectnanox.checknanoxscreen.learnMore',
+    defaultMessage: '!!!Learn more about using Yoroi with Ledger',
+  },
+})
+
+// TODO
+const url = '' // 'https://yoroi-wallet.com/...'
```

Comment:
Why is both this and `ledgerMessages.internetConnected` commented out?

Reply:
@SebastienGllmt `url` should be set once we have a video ready, we said we will add it once the video is ready.

About the `ledgerMessages.internetConnected` string, would you agree if we just remove it? I think I initially added it because we had it in the extension, but in the extension you really need internet to communicate with the http bridge. In mobile you don't strictly need internet.